go.mod

```diff
@@ -13,10 +13,10 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/pflag v1.0.5
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.29.4
+	k8s.io/api v0.29.5
 	k8s.io/apiextensions-apiserver v0.29.4
```

Comment:
its a bit annoying why this is overlooked

Reply:
Seems to be fine now.